charts/jobs/templates/cronjob.yaml

```diff
@@ -21,7 +21,7 @@ spec:
           containers:
             - image: "{{ $spec.image.repository }}:{{ $spec.image.tag }}"
               imagePullPolicy: "{{ $spec.image.pullPolicy }}"
-              name: {{ $name }}
+              name: "{{ $spec.name }}"
```

Comment:
why the quotes if we know we don't need them (looking at the schema for container names)?

Reply:
I think you are trying to cover for empty string, but those are invalid, and will be caught by validator. Please understand these distinctions and don't solve in one domain what should be solved in another.

Reply:
I think I just copied them from `job.yaml` for consistency.

Reply:
Sorry, I don't remember adding these quotes but it looks like I did. I didn't have any special reason for it.